pkg/csi/recover/recover.go

```diff
@@ -53,19 +59,29 @@ type containerStat struct {
 	startAt       metav1.Time
 }
 
-func NewFuseRecoder(kubeClient client.Client, kubeletClient *kubelet.KubeletClient, recorder record.EventRecorder) *FuseRecover {
+func NewFuseRecover(kubeClient client.Client, kubeletClient *kubelet.KubeletClient, recorder record.EventRecorder, recoverFusePeriod int) *FuseRecover {
 	return &FuseRecover{
 		SafeFormatAndMount: mount.SafeFormatAndMount{
 			Interface: mount.New(""),
 			Exec:      k8sexec.New(),
 		},
-		KubeClient:    kubeClient,
-		KubeletClient: kubeletClient,
-		Recorder:      recorder,
-		containers:    make(map[string]*containerStat),
+		KubeClient:        kubeClient,
+		KubeletClient:     kubeletClient,
+		Recorder:          recorder,
+		containers:        make(map[string]*containerStat),
+		recoverFusePeriod: recoverFusePeriod,
 	}
 }
 
+func (r *FuseRecover) Start(ctx context.Context) error {
+	// do recovering at beginning
+	// recover set containerStat in memory, it's none when start
+	r.Recover()
+	r.Run(r.recoverFusePeriod, wait.NeverStop)
```

Comment:
Since `recoverFusePeriod` is set in struct FuseRecover, `r.Run(period int, stopCh <-chan struct{})` can simplify to not deliver paramter `period` and get it from `r.recoverFusePeriod` instead.

Reply:
Thx for the suggestion. Updated.